### ISSUE TITLE
feat: show related questions on question page

### ISSUE THREAD

**Issue by drodil** [drodil/backstage-plugin-qeta#115]
Create a list (like the hot questions, etc. on the front page) and show it at question page. The list should contain related questions that have either similar tags, entities or title/description. This requires also changes to backend to get proper list out.

**Comment by drodil** [comment]
Actually, this could also be shown in the ask question form and update the list when the user changes the inputs. But might be a different issue.

**Comment by RomanSaveljev** [comment]
> Actually, this could also be shown in the ask question form and update the list when the user changes the inputs. But might be a different issue.

To me this is a no-brainer feature - why should I spend time posting question, if I know it was already answered. I would rather stop typing it. StackOverflow implemented it like this and it is a useful feature

**Comment by drodil** [comment]
Yes, it's a no-brainer feature that has waited time to implement it for over a year now. If you have some extra time at your hands, feel free to contribute 🤠
